flow/layers/child_scene_layer.h

```diff
@@ -27,16 +27,19 @@ class ChildSceneLayer : public Layer {
 
   void set_scene_token(uint32_t scene_token) { scene_token_ = scene_token; }
 
+  void set_prune(bool prune) { prune_ = prune; }
```

Comment:
> The difference between this flag and "invisible" is that it prevents children from being hit. Invisible just prevents this node specifically from being hit.

Thanks for clearing that up. Assumed that sublayers of invisible layers would be ineligible for hit-testing by default.